python/cudf/cudf/tests/test_json.py

```diff
@@ -936,14 +936,35 @@ def test_json_dtypes_nested_data():
         """,
         ),
     ],
+    # TODO failing test cases
 )
-def test_order_nested_json_reader(tag, data):
-    expected = cudf.read_json(StringIO(data), engine="pandas", lines=True)
-    target = cudf.read_json(
-        StringIO(data), engine="cudf_experimental", lines=True
-    )
+class TestNestedJsonReaderCommon:
+    @pytest.mark.parametrize("chunk_size", [10, 100, 1024, 1024 * 1024])
+    def test_chunked_nested_json_reader(self, tag, data, chunk_size):
+        expected = pd.read_json(StringIO(data), lines=True)
```

Comment:
Just a nitpick, but it seems like we might want to compare to a non-chunked `cudf.read_json` read, rather than to `pd.read_json` (at least that *seems* to be the purpose of the test to me).

Reply:
That's right. updated.